src/protocols/bitcoin/views/AccountDetails.vue

```diff
@@ -1,19 +1,27 @@
 <template>
   <AccountDetailsBase class="account-details">
+    <template #account-info>
+      <AccountInfo
+        :address="activeAccount.address"
+        :name="activeAccount.name"
+        :idx="activeAccount.idx"
+        :protocol="activeAccount.protocol"
+        can-copy-address
+        with-protocol-icon
+      />
+    </template>
+
     <template #balance>
-      <div class="text-center">
-        <h3>BTC</h3>
-        <p>This is a temporary component that uses the account base boilerplate.</p>
-      </div>
+      <BalanceInfo
+        :balance="balanceNumeric"
+        :protocol="activeAccount.protocol"
+        horizontal-offline-message
+      />
     </template>
 
     <template #buttons>
-      <BtnBox
-        text="Example button 1"
-      />
-      <BtnBox
-        text="Example button 2"
-      />
+      <OpenTransferReceiveModalButton />
+      <OpenTransferSendModalButton />
```

Comment:
I think those could be handled by the base component as the logic is handled elsewhere